apps/gallery/js/ImageEditor.js

```diff
@@ -1656,6 +1645,22 @@ ImageEditor.prototype.getCropRegion = function() {
   };
 };
 
+// Covert the cropped region in image pixels to object with left, right, top,
+// bottom properties in CSS Pixel to be used to show crop overlay
+ImageEditor.prototype.getCropOverlayRegion = function(region) {
```

Comment:
This would be clearer, I think, if you renamed the region argument to 'cropSettings'. Since the argument comes from editSettings.crop, this new name helps to distinguish it from the the region that is the return value.